### PR TITLE
New blog time! beauty.md is "Beautifying Your Terminal - Custom CLI Commands"

### PR DESCRIPTION
**Pull request by browner1903**

### CODE REVIEW

File: posts/beauty.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ |___/             |_____|             |_____|
+```
+
+While `figlet` is a cool,easy-to-use command to get some really quick ASCII designs, it is only limited to text. You can use some other resources to get pictures/graphics ASCII-ified, such as [manytools.org](https://manytools.org/hacker-tools/convert-images-to-ascii-art/). Here, you can take a certain logo that you like, edit how large or small it is, and you have your very own text-ified logo, like you would see in some cool hacker movie/videogame. More on that later...
+
+## lolcat
+
+With the `lolcat` command, you can make your text into a rainbow! For example, you can type `lolcat file/path` and it will return that file's contents in rainbow style! So, if I were to (for some reason) have an ASCII art of Hello Kitty in my system, it would look something like this:
+
+[!ascii kitty being rainbow-coloured in my terminal, thanks to lolcat. ](/res/ascii-kitty.png)
```
Comment (DistroByte) [inline posts/beauty.md:30]:
```suggestion
![ascii kitty being rainbow-coloured in my terminal, thanks to lolcat. ](/res/ascii-kitty.png)
```

Should actually embed the image this way 😉

File: posts/beauty.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+##markdown
```
Comment (DistroByte) [inline posts/beauty.md:1]:
```suggestion
```

File: posts/beauty.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+Hey Brickies!
+
+As computing students, we all know that the best way to do anything on a PC is via the command line (how else would people know that you're in IT?!) And as Redbrick members, you are aware that you have a free to use wetty environemnt? (wetty.redbrick.dcu.ie)
```
Comment (DistroByte) [inline posts/beauty.md:5]:
```suggestion
As computing students, we all know that the best way to do anything on a PC is via the command line (how else would people know that you're in IT?!) And as Redbrick members, you are aware that you have a free to use wetty environemnt? ([wetty.redbrick.dcu.ie](https://wetty.redbrick.dcu.ie))
```

File: posts/beauty.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+## A Word on custom/changing bash commands
+
+Now, while some of these suggestions do not help you out in terms of productivity, let's have a look at one that can actually help you navigate your CLI a little better.
+
+By editing the .bashrc file, we can add our own custom `alias` to commands, to make the terminal easier to use. Let's say you are sick of typing out the same commands that you use frequently in your CLI, such as `ls -a`. This command lists all items in a current directory, including hidden `.filename` files. But typing out l, then s, then -, then a, AND a space in between them?! Don't be ridiculous, that's just too much work, especially if you use this command over and over. Let's shorten that in a memorable but functional way. `ls -a` can be shortened down to `la` by using the command: 
+
+`alias la="ls -a"`
+
+Add this command to the BOTTOM of your .bashrc file (type `cd`, press enter, then type `nano .bashrc` and scroll to the bottom of the file, then add it in there.) After you add it to .bashrc, CtrlS, CtrlX and give it a shot!
```
Comment (DistroByte) [inline posts/beauty.md:58]:
```suggestion
Add this command to the BOTTOM of your .bashrc file (type `cd`, press enter, then type `nano .bashrc` and scroll to the bottom of the file, then add it in there.) After you add it to `~/.bashrc`, CtrlS, CtrlX and give it a shot! 
```

**Review by DistroByte — COMMENTED** [review]
Looks awesome! Some suggestions on formatting etc

**Review by browner1903 — COMMENTED** [review]
If I missed one do let me know, but thanks for suggestion. Ill read through once more then let you know?

File: posts/beauty.md (hunk 3 above)
Comment (d-fens) [inline posts/beauty.md:5]: environemnt - typo

**Review by wizzdom — APPROVED** [review]
LGTM

**Review by DistroByte — APPROVED** [review]
LGTM 🚀

**Pull request merged**